shell/version/version.cc

```diff
@@ -1,7 +1,8 @@
 // Copyright 2013 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
-// FLUTTER_NOLINT
+
+// FLUTTER_NOLINT: https://github.com/flutter/flutter/issues/68332
```

Comment:
Can we just use one meta-issue with a checklist for these?

Reply:
Not entirely opposed to it but I'm concerned that that has a lot of the same issues as not having a link at all; people are tempted to just copy-paste without investigating further. This would also increase the likelihood of adding the NOLINT but forgetting to add it to the bug.

Part of the goal here is to make it painful to add new opt-outs by accurately describing the issue.